Mac-App/Backend/Backend/BackendAPI/Middleware/DependencyGraphSideEffects/DependencyGraphSideEffects.swift

```diff
@@ -18,7 +18,7 @@ func dependencyGraphSideEffects(parser: @escaping SwiftDepsParserType) -> Middle
 
                 switch action {
                 case let DependencyGraphAction.mapFrom(swiftDeps):
-                    dispatchAsyncOnBackendQueue {
+                    dispatchAsyncOnConcurrentBackendQueue {
```

Comment:
is Backend really needed in the name of the method? Or can it be replaced by Background?

Reply:
Hmm you're right, it's already in the `Backend` framework so it is implicit. Also, this function is not even available outside the module. Will rename 👍

Reply:
https://github.com/Thurman1776-/Uncomplect-us/pull/28/commits/16f7a12d6e39e5782de97cb86a902df2f38af5c4